game.h

```diff
@@ -9,11 +9,13 @@ class Game{
 public:
     Game(int fieldSize = 5);
     const void startGame(int fieldSize = 5);
-    void endGame(std::string condition);
+    void endGame();
     int getSize(){return gameField.getSize();}
     std::vector<std::vector<Cell>> getFieldCells(){return gameField.getField();}
     Score& getScore(){return gameScore;}
     void moveCellsOnField(Direction direction);
+    const bool isLose(){return loseStatus;}
```

Comment:
no need of const bool, the method itself should be const, check how to make a const method, not a return value

Reply:
done